cpp/finite-element.h

```diff
@@ -215,9 +216,29 @@ class FiniteElement
   /// @return The family name
   const std::string& family_name() const;
 
-  /// Get the mapping used for this element
+  /// Get the mapping type used for this element
   /// @return The mapping
-  const std::string& mapping_name() const;
+  const mapping::type mapping_type() const;
+
+  /// Map a function value from the reference to a cell
```

Comment:
Should this be in the public API header file? (`basix.h`)?

```
/// Map a function value on the reference cell to the physical cell
```
?

Reply:
It's now in both